src/v2/view-builder/views/consent/ConsentViewHeader.js

```diff
@@ -0,0 +1,31 @@
+import { View, _ } from 'okta';
+
+import consentLogoHeaderTemplate from '../../../../views/shared/templates/consentLogoHeaderTemplate';
+
+const defaultLogo = '/img/logos/default.png';
+
+const ConsentViewHeader = View.extend({
+  className: 'consent-title detail-row',
+  template: consentLogoHeaderTemplate,
```

Comment:
i would put the template here instead.

Reply:
https://github.com/okta/okta-signin-widget/pull/1637/commits/ce2298ed42df1b22be3dca6cfb4e897b19aefe63